src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -2584,3 +2584,20 @@ void QgsCoordinateReferenceSystem::invalidateCache( bool disableCache )
   }
   sCrsStringLock()->unlock();
 }
+
+bool operator> ( const QgsCoordinateReferenceSystem &c1, const QgsCoordinateReferenceSystem &c2 )
+{
+  return c1.d->mSrsId > c2.d->mSrsId;
```

Comment:
We aren't always guaranteed to have a srs id... Is the equality operator (which compares wkt) used for maps in conjunction with this?

Reply:
Then let's just switch these operators to the wkt. The equality operator might be used too, but it's not sufficient.